package.json

```diff
@@ -262,7 +262,7 @@
     "classnames": "^2.2.3",
     "prop-types": "^15.5.8",
     "react-popper": "^1.3.6",
-    "react-transition-group": "^2.3.1"
+    "react-transition-group": "^4.4.1"
```

Comment:
Just realized that the latest RTG requires a higher peer dependency of react 16.6, whereas reactstrap is at 16.3.  This would have to be considered a breaking change.

Reply:
Should I pick a version of RTG which doesn't require anything higher than 16.3, or should I go through the process of bumping the major version of Reactstrap?

Reply:
I believe only 4.4.x resolves the issues with findDOMNode, so we're stuck there.  Will need to get @TheSharpieOne's feedback here on major version bumps.

Reply:
major version bumps are fine. But we should try to group all breaking changes. I know there are other breaking changes that could be made (such as forward refs).

Reply:
Should I bump the major version in this PR, or is this something to be done in another PR?